tests/e2e/pact/services/convert-example-fetcher.ts

```diff
@@ -0,0 +1,47 @@
+import path from "path"
+import { ConvertCase } from "../models/cases/convert-case"
+import { exampleFiles } from "./example-files-fetcher"
+
+const examplesRootPath = "../resources/parent-prescription"
+
+const convertResponseFiles = exampleFiles.filter(exampleFile => exampleFile.isResponse && exampleFile.endpoint === "convert")
+
+const convertRequestFiles = exampleFiles.filter(exampleFile =>
```

Comment:
This filter is confusing to read at first. Not urgent but could be cleaned up in a separate ticket to make it more readable?

Reply:
Sounds good